lib/classes/calculator_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+class Calculator
+  
+  def add(list_of_numbers)
+    list_of_numbers.sum
+  end
+  
+  def max(list_of_numbers)
+    list_of_numbers.max
+  end
+  
+  def min(list_of_numbers)
+    list_of_numbers.min
+  end
+  
+  def odd(list_of_numbers)
+    list_of_numbers.sum.odd?
+  end
+  
+  def even(list_of_numbers)
+    list_of_numbers.sum.even?
+  end
+  
+  def self.hello
+   "Hello, I am your Calculator :)"
+  end
+end
```

Comment:
this should not be inside spec file, 
please move that inside  lib/classes/calculator.rb

definition of class should be inside one file only, due to reason that it is big chance that it will be used over multiple places in code base.

e.g. class Calculator inside lib/classes/calculator.rb
some other class e.g. Person inside lib/classes/person.rb

and when you want to use it then you can use require 'lib/classes/calculator.rb'